biodatasets/bio_simlex/bio_simlex.py

```diff
@@ -0,0 +1,172 @@
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    BIGBIO_VERSION = datasets.Version(_BIGBIO_VERSION)
+
+    BUILDER_CONFIGS = [
+        BigBioConfig(
+            name="bio_simlex_source",
+            version=SOURCE_VERSION,
+            description="bio_simlex source schema",
+            schema="source",
+            subset_id="bio_simlex",
+        ),
+        BigBioConfig(
+            name="bio_simlex_bigbio_pairs",
+            version=BIGBIO_VERSION,
+            description="bio_simlex BigBio schema",
+            schema="bigbio_pairs",
+            subset_id="bio_simlex",
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = "bio_simlex_source"
+
+    def _info(self) -> datasets.DatasetInfo:
+
+        if self.config.schema == "source":
+            features = datasets.Features(
+                {
+                    "text_1": datasets.Value("string"),
+                    "text_2": datasets.Value("string"),
+                    "score": datasets.Value("string"),
+                }
+            )
+
+        elif self.config.schema == "bigbio_pairs":
+            features = schemas.pairs_features
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager) -> List[datasets.SplitGenerator]:
+        """Returns SplitGenerators."""
+
+        url = _URLS[_DATASETNAME]
+        data_dir = dl_manager.download_and_extract(url)
+
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                # Whatever you put in gen_kwargs will be passed to _generate_examples
+                gen_kwargs={
+                    "filepath": data_dir,
+                    "split": "train",
+                },
+            ),
+        ]
+
+    def _generate_examples(self, filepath, split: str) -> Tuple[int, Dict]:
+        """Yields examples as (key, example) tuples."""
+        with open(filepath, "r", encoding="utf-8") as f:
+            for id_, line in enumerate(f):
+                word1, word2, score = line.split("\t")
+                if self.config.schema == "source":
+                    yield id_, {
+                        "text_1": word1,
+                        "text_2": word2,
+                        "score": str(score),
+                    }
+
+                elif self.config.schema == "bigbio_pairs":
+                    yield id_, {
+                        "id": str(id_),
+                        "document_id": "NULL",
+                        "text_1": word1,
+                        "text_2": word2,
+                        "label": str(score),
+                    }
+
+
+# This allows you to run your dataloader with `python bio_simlex.py` during development
+# TODO: Remove this before making your PR
+if __name__ == "__main__":
+    datasets.load_dataset(__file__)
```

Comment:
remove please.

Reply:
@galtay @tanmaylaud I manually removed the dataset checker above; if you're still treating this as a draft PR please feel free to replace it but remove it before making the final PR.